src/Sql/Sql/Auditing/Cmdlet/SetAzSqlServerAuditing.cs

```diff
@@ -0,0 +1,374 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Auditing.Model;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.Server.Model;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
+{
+    /// <summary>
+    /// Sets the auditing settings properties for a specific database server.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Set,
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlServerAuditing",
+        SupportsShouldProcess = true),
+        OutputType(typeof(ServerBlobAuditingSettingsModel))]
+    public class SetAzSqlServerAuditing : SqlServerAuditingSettingsCmdletBase
```

Comment:
Add required `DefaultParameterSetName` property to the `Cmdlet` attribute. This may also fix the issue with Static Analysis.

Reply:
Done, did not help with static analysis.